docs/DeveloperGuide.adoc

```diff
@@ -295,88 +295,172 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |Finance Staff |keep track of all the worker's details | ensure proper financial accounting for the company
+
+|`* * *` |Accountant |see the payslip of all employees| easily track expenses related to employee wages
+
+|`* * *` |Accountant |access the employee's personal details |keep track of all the worker's details
+
+|`* * *` |Employee of a Company |able to see payslip| for documenting purposes
+
+|`* * *` |Admin Staff |be able to track the employee's paid and medical leave| ensure the workers are not abusing the leave system
+
+|`* * *` |Admin Staff |view my company's available manpower| decide on the manpower allocation
+
+|`* *` |Employee |view upcoming events | record them on my calendar
+
+|`* *` |Admin Staff|find upcoming events | plan on the logistics needed ahead of time
+
+|`* *` |Manager|tag my employees with comments/remarks | have better manpower allocation for events
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
-_{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `AddMin+` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== Use case: Delete employee
+
+*MSS*
+
+1.  User requests to list employees
+2.  AddMin+ shows a list of employees
+3.  User requests to delete a specific employee by stating the Employee ID
+4.  AddMin+ deletes the employee
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. The given Employee ID is invalid.
++
+[none]
+** 3a1. AddMin+ shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Edit Event in Event List
+
+*MSS*
+
+1.  User requests to view the company’s current list of events.
+2.  AddMin+ shows a list of events, with each event containing a unique Event ID.
+3.  User specifies Event ID, and list the details of the event to edit
+4.  AddMin+ edits and saves the updated event details.
++
+Use case ends.
+
+*Extensions*
+
+* 3a. Event ID given is invalid.
++
+[none]
+** 3a1.AddMin+ shows an error message and refuses the edit.
++
+Use case resumes at step 2.
++
+* 3b. No details to edit were specified, or is of the wrong input format
++
+[none]
+** 3b1. AddMin+ displays an error message warning the user that no details were edited.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Allocate Manpower to Event manually
+
+*MSS*
+
+1.  User requests to view the company’s current list of employees and events.
+2.  AddMin+ shows the list of employees and events.
+3.  User states an Employee_ID that is matched to an Event_ID
```

Comment:
It would be better if you could be more consistent with regards to using full stops at the end of sentences